back/app/controllers/web_api/v1/cosponsors_initiatives_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class WebApi::V1::CosponsorsInitiativesController < ApplicationController
+  def accept_invite
+    @cosponsors_initiative = CosponsorsInitiative.find(params[:id])
+    authorize @cosponsors_initiative, :accept_invite?
```

Comment:
```suggestion
    authorize @cosponsors_initiative
```
Isn't it enough? But basically, it's the same.

Reply:
Not sure. I copied from `:block?` for user blocking, but I'll check if it's needed.